dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05200WorkflowTaskUniqueKey.java

```diff
@@ -36,6 +41,37 @@ public void executeUpgrade() throws DotDataException, DotRuntimeException {
             throw new DotDataException(e.getMessage(), e);
         }
         final DotConnect dotConnect = new DotConnect();
+        
+        List<Map<String,Object>> results = dotConnect.setSQL("select webasset,language_id from workflow_task group by webasset,language_id having count(*)>1").loadObjectResults();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'results' could be declared final](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=4666366)